local/main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	hookdeck "github.com/hookdeck/hookdeck-go-sdk"
+	hookdeckclient "github.com/hookdeck/hookdeck-go-sdk/client"
```

Comment:
also curious if there's a reason you use `hookdeckclient` instead of `hookdeckClient`. Honestly just trying to learn and see if this is more conventional from Go community?

Reply:
This is actually a Go idiom - all package names should be all lowercase ([link](https://go.dev/blog/package-names)). The `hookdeckclient` package alias is a little longer than we'd normally like, but it prevents conflicts with a likely `client` variable.